t/op/taint.t

```diff
@@ -1565,7 +1565,7 @@ SKIP: {
         }
 
         skip "SysV shared memory operation failed", 1 unless 
-          $rcvd eq $sent;
+          ($rcvd // '') eq $sent;
```

Comment:
this is correct, better read IMO like `unless defined $rcvd && $rcvd eq $sent`
setting `$rcvd` to an empty string will solve some issues but not all and is not a good solution

Reply:
> this is correct, better read IMO like `unless defined $rcvd && $rcvd eq $sent`
> setting `$rcvd` to an empty string will solve some issues but not all and is not a good solution

Updated.